src/appmixer/leadspicker/README.md

```diff
@@ -0,0 +1,5 @@
+```bash

```

Comment:
Maybe add some headers to the readme to keep it in line with other readmes for generated connectors?

```suggestion
# Leadspicker

## Generate Connector

```bash
```